src/core/execute.c

```diff
@@ -2250,6 +2250,7 @@ static int setup_exec_directory(
                 uid_t uid,
```

Comment:
hmm, i am a bit confused by the commit msg. the symlink is the other way: /var/lib/foo → /var/lib/private/foo?

what do you mean by ephemeral /var/lib? what's that supposed to be?